src/ebonite/ext/flask/server.py

```diff
@@ -27,6 +22,7 @@ def create_executor_function(interface: Interface, method: str):
     :param method: method name
     :return: callable view function
     """
+    from flask import g, jsonify, request, send_file
```

Comment:
why localize imports?

Reply:
To require Flask/flasgger installation inside runtime (images) only and not during work with metadata (in Jupyter notebooks, etc). This is similar to sklearn model wrapper requiring sklearn installation when model is actually called only.

Reply:
you mean to be able to build images with flask server even if you dont have flask installed?

Reply:
ok for now, but maybe we can do better here

Reply:
Yes, I mean building Flask images without Flask installation. I agree that imports are kind of dumb solution. To do better than this we should decompose server runtime (how to expose model endpoints) and metadata (how to build image with server) parts for all servers. I decided this to be quite a cruel change in scope of this PR. But this could certainly be taken into account in future.